### ISSUE TITLE
"node.debug is not a function"

### ISSUE THREAD

**Issue by RichyRich42** [586837r/node-red-contrib-alexa-remote2#123]
Hi,

I have connection issues with alexa-remote2. 
I can not connect via Proxy, Cookie or usr/pwd, however I got alexa-remote (version1) to work out of the box.
I suspect it is because of the error: "node.debug is not a function" and then afterwards I get "account not initialised"
I run node v10.20.0 on a raspberry in my private LAN

**Comment by 586837r** [comment]
What Node-RED version are you running?

**Comment by RichyRich42** [comment]
Node red v0.16.2

**Comment by 586837r** [comment]
Yeah that's a really old version. Updating Node-RED will fix that.
